config.js.sample

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    secret: 'Create**Ur*-Secret=+Key-'
+};
```

Comment:
@germancin Thank you for including this - made it a lot easier to fire up your project on my lappy 👍 You get a gold ⭐️